doc/manual/manual.tex

```diff
@@ -4028,9 +4028,17 @@ \subsubsection{A note on the syntax of formulas in input files}
 \texttt{x}, \texttt{y} and \texttt{z} to reference coordinates (unless a
 particular plugin uses different variables, such as the depth), the symbol
 \texttt{t} for time in many situations, and allows you to use all of the
-typical mathematical functions such as sine and cosine. Another common case is
-an if-statement that has the general form
-\texttt{if(condition, true-expression, false-expression)}. For more examples of
+typical mathematical functions such as sine and cosine. Based on the muparser 
+library, deal.II supports additional functions, include \texttt{|}, \texttt{\&}, 
+\texttt{int()}, \texttt{ceil()}, \texttt{floor()}, \texttt{cot()}, 
+\texttt{csc()}, \texttt{sec()}, \texttt{pow()}, \texttt{log()}, 
+\texttt{erfc()}, \texttt{rand()}, and \texttt{rand\_seed()}. 
+For more detailed information, see 
+\url{http://www.dealii.org/developer/doxygen/deal.II/classFunctionParser.html#details}. 
+A common case is an if-statement that has the general form
+\texttt{if(condition, true-expression, false-expression)}. As an example, 
+the statement "if 1<x<4, output 1; else, output 0" can be expressed as 
```

Comment:
If you run this through latex, this is going to look funny. First, you need to use `\`\`` as opening quotes and `"` as closing ones. Second, you probably want to enclose the expression `1<x<4` into dollar signs so that it is rendered as a formula -- the `<` character leads to a weird symbol in regular text mode.